### PR TITLE
Rename homeManagerModules to homeModules

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

**Review by grahamc — APPROVED** [review]

**Review by grahamc — CHANGES_REQUESTED** [review]
x

File: flake.nix [hunk 1]
```diff
@@ -126,7 +126,7 @@
         };
       };
 
-      homeManagerModules.default = { lib, config, pkgs, ... }: {
+      homeModules.default = { lib, config, pkgs, ... }: {
```
Comment (grahamc) [inline flake.nix:129]: I was too quick. We've got folks using the module as-is, let's add a little help for our friends:

```suggestion
      homeManagerModules.default = self.homeModules.default;
      homeModules.default = { lib, config, pkgs, ... }: {
```

(Note I haven't tested this, so please make sure it works ...)

Comment (lucperkins) [reply flake.nix:129]: Good call. Done.

**Review by grahamc — APPROVED** [review]

File: flake.nix [hunk 2]
```diff
@@ -150,5 +150,9 @@
           };
         };
       };
+
+      # homeModules is preferred but this ensures compatibility for anyone
+      # currently using nix.homeManagerModules.default in their config
+      homeManagerModules.default = self.homeModules.default;
```
Comment (cole-h) [inline flake.nix:156]: Were you able to test this to make sure it behaves how you'd expect?

Comment (lucperkins) [reply flake.nix:156]: Yes, I ran the test locally with `modules = [ nix.homeManagerModules.default ]` instead of `modules = [ nix.homeModules.default ]`

**Pull request merged**